### PR TITLE
[1489] Added settings for search api on manage backend

### PR DESCRIPTION
**Pull request by defong**
### Context
Manage backend and search and compare communication and credentials

### Changes proposed in this pull request
Added search and compare communication and credentials to manage backend

### Guidance to review
Similar to `SETTINGS__MANAGE_API__SECRET`

related to 
https://github.com/DFE-Digital/bat-infrastructure/pull/132

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[[M] Sync course to search-and-compare](https://trello.com/c/wXvgCtNG/1489-m-sync-course-to-search-and-compare)

File: config/settings/qa.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 manage_api:
   base_url: https://bat-dev-manage-courses-api-app.azurewebsites.net
   secret: please_change_me
+manage_api:
```
Comment (smneal) [inline config/settings/qa.yml]: Should this be?
```suggestion
search_api:
```

File: config/settings/staging.yml [hunk 2]
```diff
@@ -1,3 +1,6 @@
 manage_api:
   base_url: https://bat-staging-manage-courses-api-app.azurewebsites.net
   secret: please_change_me
+manage_api:
```
Comment (smneal) [inline config/settings/staging.yml]: Should this be?

```suggestion
search_api:
```

**Comment by smneal** [comment]
@defong The release definition will need updating just prior to this being merged.

**Pull request merged**